src/app/(home)/page.tsx

```diff
@@ -1,3 +1,78 @@
+import styles from "./page.module.css";
+import classNames from "classnames";
+import Button from "@codegouvfr/react-dsfr/Button";
+import Tile from "@codegouvfr/react-dsfr/Tile";
+import Image from "next/image";
+
 export default async function HomePage() {
-  return <>TODO</>;
+    return <div>
+        <div className={classNames(styles.section, styles.presentationSection)}>
+            <div className="fr-container">
+                <h1 className={styles.mainTitle}>{"La plateforme d'accompagnement sur mesure pour vos projets"}</h1>
+                <div
+                    className="fr-text--lead">{"Accélérer vos projets du quotidien en les orientant le plus en amont possible vers le bon service d'accompagnement"}
+                </div>
+                <Button className={styles.accompagnementButton} linkProps={{href: '#'}}>Etre accompagné</Button>
+            </div>
+        </div>
+        <div className={classNames(styles.section)}>
+            <div className="fr-container">
+                <h2 className={classNames("fr-h2")}>{"Un service intégré à l'écosystème"}</h2>
```

Comment:
Pas sur qu'il y ait besoin de mettre la class fr-h2 sur un h2 ?
```suggestion
                <h2 className="fr-h2">Un service intégré à l'écosystème</h2>
```

Reply:
Si je l'enlève, le titre sera en bleu du fait du fichier globals.css.
Si on peut retirer ce qu'il y a dans le globals.css pour les hx, alors cette classe sera effectivement inutile.